src/main/webapp/step2.html

```diff
@@ -1,9 +1,14 @@
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>Choose transport</title>
-    <link rel="stylesheet" href="style.css">
-  </head>
-  <body>
-  </body>
+    <head>
+        <meta charset="UTF-8">
+        <title>Choose transport</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+        <h1>STEP 2</h1>
```

Comment:
optional: Throughout can specify "step 2 of n" so that people are less scared ;) 

Doesn't have to be in this change though.

Reply:
I understand. Thanks!